components/simulation-form/InterventionPeriodSection.tsx

```diff
@@ -1,8 +1,9 @@
+import {input} from '@covid-modeling/api'
+import {Intensity} from '@covid-modeling/api/dist/src/model-input'
```

Comment:
Should this still use `dist`? Don't we expose `Intensity` in `input`?

Reply:
Thanks that should change.